.typos.toml

```diff
@@ -33,6 +33,7 @@ extend-ignore-re = [
 ]
 
 [default.extend-identifiers]
+"advices" = "advices" # typo in version_check.go
```

Comment:
Why not to fix typo instead? Or we have there singular and plural `advise`?

Reply:
This is the variable name: array of multiple advice. But can change it to something like advs.

Reply:
How about this? https://github.com/percona/percona-toolkit/pull/801

Reply:
Looks good.

Reply:
Merged